src/main/java/handlers/HandlerGetFileContents.java

```diff
@@ -4,25 +4,24 @@
 
 public class HandlerGetFileContents implements Ihandler {
   private IFileIO fileIO;
-  private ResponseHeadersForContent responseHeaders;
+  private ContentTypeForFileExtension contentType;
```

Comment:
I agree, although for a different reason: to me, it feels more reasonable for these particular handlers to just know about `ContentTypeForFileExtension` than for the top-level `HandlerRouter` to have to know about such a minute detail and provide it to the handlers.